frontend/src/pages/AccountSend.vue

```diff
@@ -600,63 +600,56 @@ function useSendForm() {
 
   // Batch Send Computed Form Parameters
   const batchSendHumanTotalAmount = computed(() => {
-    let ethTotal;
+    let ethSendAmount: BigNumber | undefined;
```

Comment:
I think the types can be removed here

Reply:
Sounds good, removed here: https://github.com/ScopeLift/umbra-protocol/pull/608/commits/b1309b86c0f4b92fb444cef47d970b4a6a93e213